src/components/LoadingState/LoadingState.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Box, CircularProgress } from '@mui/material';
+
+import * as Styles from './styles';
+
+export default function LoadingState() {
+  return (
+    <Styles.Container>
```

Comment:
**suggestion:** thinking in future implementation, what do you think to add a `aria-label` to help to test the component?

```suggestion
    <Styles.Container aria-label="a-good-name">
```